### PR TITLE
proxsuite: 0.3.5-1 in 'rolling/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by wxmerkt**
Increasing version of package(s) in repository `proxsuite` to `0.3.5-1`:

- upstream repository: https://github.com/Simple-Robotics/proxsuite.git
- release repository: https://github.com/ros2-gbp/proxsuite-release.git
- distro file: `rolling/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

### CODE REVIEW

**Comment by clalancette** [comment]
Held for an upcoming [Rolling sync](https://discourse.ros.org/t/preparing-for-rolling-sync-2023-03-14/30176)

**Comment by wep21** [comment]
@wxmerkt I appreciate it if you could also do bloom-release to `humble`.

**Comment by wxmerkt** [comment]
@wep21 Of course, will do. I want to make sure that we don't need patches and that it builds / works successfully on Rolling first before releasing to more distros

File: rolling/distribution.yaml [hunk 1]
```diff
@@ -3338,6 +3338,21 @@ repositories:
       url: https://github.com/mrpt-ros-pkg/pose_cov_ops.git
       version: master
     status: maintained
+  proxsuite:
+    doc:
+      type: git
+      url: https://github.com/Simple-Robotics/proxsuite.git
+      version: main
```
[the remark this answers is not in the document]
Comment (wxmerkt) [reply rolling/distribution.yaml]: Hi @nuclearsandwich, this was intended but I may have got this wrong. We are releasing off the `main` branch but the latest development branch is `devel`

**Comment by wep21** [comment]
> Of course, will do. I want to make sure that we don't need patches and that it builds / works successfully on Rolling first before releasing to more distros

@wxmerkt I can download `ros-rolling-proxsuite` from ros2-testing repo and find target by `find_package` in CMakeLists.txt. Is it possible to do bloom-release to `humble`?

**Comment by wxmerkt** [comment]
Thank you for testing/verifying and the ping @wep21 - here's the release for humble: https://github.com/ros/rosdistro/pull/36648

**Pull request merged**